### DISCUSSION TITLE
Foster StackOverflow and give answers to the most visited questions regarding maven based on Maven 4 and new documentation

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#10] [Ideas for Funding]
StackOverflow has a lot of questions regarding Maven. For several questions the answers are outdated. We should check the Maven Q&As at StackOverflow and provide up to date answers based on new documentation (that is proposed by other milestones)

**Comment by slawekjaranowski** [comment]
As it is mentioned as epic which will be reported as spending fundings, in another place we talk about progress in epics ... so how we track progres, state and son in epic like it?

**Comment by ascheman** [comment]
This discussion has been converted to Epic #60 for tracking.

Closing this discussion.
